.github/workflows/publish-to-test-pypi.yaml

```diff
@@ -0,0 +1,49 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on: [push, pull_request]
+
+jobs:
+  build-n-publish:
+    name: Build and publish to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        # Adding -l {0} helps ensure conda can be found properly.
+        shell: bash -l {0}
+    env:
+      ENV_NAME: publish
+      PYTHON: 3.8
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          miniconda-version: "latest"
+          python-version: ${{ env.PYTHON }}
+          environment-file: ci/${{ env.ENV_NAME }}.yaml
+          activate-environment: ${{ env.ENV_NAME }}
+
+      - name: Conda Info
+        run: |
+          conda info -a
+          conda list
+          PYVER=`python -c "import sys; print('{:d}.{:d}'.format(sys.version_info.major, sys.version_info.minor))"`
+          if [[ $PYVER != ${{ env.PYTHON }} ]]; then
+            exit 1;
+          fi
+
+
+      - name: Build a binary wheel and a source tarball
+        run: |
+          python -m pep517.build --source --out-dir dist/ .
```

Comment:
apparently you should just use `python -m build .` these days

Reply:
done